topic-operator/src/main/java/io/strimzi/operator/topic/v2/BatchingLoop.java

```diff
@@ -246,20 +249,20 @@ private void fillBatch(int batchId, Batch batch) throws InterruptedException {
             final long deadlineNanoTime = System.nanoTime() + maxBatchLingerMs * 1_000_000;
             while (true) {
                 if (batch.size() >= maxBatchSize) {
-                    LOGGER.traceOp("[Batch #{}] reached maxBatchSize, batch complete", batchId, maxBatchSize);
+                    LOGGER.traceOp("[Batch #{}] Reached maxBatchSize, batch complete", batchId, maxBatchSize);
                     break;
                 }
 
                 long timeoutNs = deadlineNanoTime - System.nanoTime();
                 if (timeoutNs <= 0) {
-                    LOGGER.traceOp("[Batch #{}] {}ms linger expired", batchId, maxBatchLingerMs);
+                    LOGGER.traceOp("[Batch #{}] {}ms Linger expired", batchId, maxBatchLingerMs);
```

Comment:
It's a bit weird to have the capital like this, `10ms Linger...` don't you think?